apps/cookbook/src/app/showcase/item-showcase/item-showcase.component.html

```diff
@@ -276,19 +276,26 @@ <h4>Rotation animation</h4>
   It is possible to trigger an animation that rotates the disclosure icon by passing a boolean flag
   to
   <code>[rotateIcon]</code>
-  .
+  . When set to
+  <code>true</code>
+  it will rotate the icon 180 degrees and when set to
+  <code>false</code>
+  it will rotate the icon back to it's original position.
 </p>
 
-<cookbook-example-viewer [html]="disclosureAnimationExample.template">
+<cookbook-example-viewer
+  [html]="disclosureAnimationExample.template"
+  [ts]="disclosureAnimationExample.animationCodeSnippetTS"
+>
   <cookbook-item-example-disclosure-animation
     #disclosureAnimationExample
   ></cookbook-item-example-disclosure-animation>
 </cookbook-example-viewer>
 
 <p class="comment small">
   <em>
-    <strong>Please note</strong>
-    , the animation of the disclosure icon is only available for:
+    <strong>Please note:</strong>
```

Comment:
Great 👌🏻 
Could you maybe change it also for the note under "Disclosure"? 🙏🏻 
![image](https://github.com/kirbydesign/designsystem/assets/9556693/e29eb5b1-0cec-4f45-bd72-88aa05917d2c)